package.json

```diff
@@ -34,12 +35,15 @@
     "rst2mdown": "^0.1.0",
     "semantic-ui-css": "^2.2.4",
     "source-map-support": "^0.4.0",
+    "start-server-and-test": "^1.11.3",
     "uglifyify": "^3.0.1",
     "uglifyjs": "^2.4.10"
   },
   "scripts": {
     "watch": "./scripts/watch",
     "start": "./scripts/get_boards && npm run serve & npm run watch",
+    "smoke-test": "start-test serve 8080 cy:run",
+    "cy:run": "cypress run --record --key 0ef4cdd6-db15-4617-a080-eeb584c686f4",
```

Comment:
Isn't that key supposed to be secret?

Reply:
Yes. Though I don't intend to leave this instance of `Cypress` live after merging this branch. Also, it will require writing a script to parse the `ENV` var; I just took the easy route.

Reply:
Now the `CYPRESS_KEY` in Travis settings holds the value of the current key.

Please, create a Cypress account and generate a new key: 
1. delete `projectId`from `cypress.json` 
2. run `npx cypress open` and choose the `Runs` tab and follow the steps there.

Then update the key in Travis settings.

Reply:
Do I need to, or is it fine running on your key for now?

Reply:
It's okay. Though keep in mind that the key is in the git log. Also, I think it's better to have one managing all resources.